templates/events/event.html

```diff
@@ -92,7 +119,7 @@ <h5>Event Status </h5>
               <p class="col-sm-9">
                 {% if post.eventStatus == "approved" %}
                     <!-- temporily change to published -->
-                  <span class="badge badge-pill badge-success"> published </span>
+                  <span class="badge badge-pill badge-success"> pending </span>
```

Comment:
Let's use warning class to match the pending style.
```suggestion
                  <span class="badge badge-pill badge-warning"> pending </span>
```